android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -115,10 +121,34 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         initializeDropdownValues();
         createDatePicker();
         setUpDateTextField();
+        createTimePicker();
         setUpRecyclerView(view);
         setUpDialog();
     }
 
+    private void createTimePicker() {
+        Calendar calendar = Calendar.getInstance();
+        int currentHour = calendar.get(Calendar.HOUR_OF_DAY);
+        int currentMinutes = calendar.get(Calendar.MINUTE);
+
+        Log.i(TAG, String.valueOf(currentHour));
+        Log.i(TAG, String.valueOf(currentMinutes));
+
+        timePicker = new MaterialTimePicker.Builder()
+                .setInputMode(MaterialTimePicker.INPUT_MODE_KEYBOARD)
+                .setTimeFormat(TimeFormat.CLOCK_24H)
+                .setHour(currentHour)
+                .setMinute(currentMinutes)
+                .setTitleText(R.string.time_message_date_picker)
+                .build();
+        timePicker.addOnPositiveButtonClickListener(dialog -> {
+            int hours = timePicker.getHour();
+            int minutes = timePicker.getMinute();
+            dateText.append(", " + hours + ":" + minutes + " Uhr");
```

Comment:
I think my previous comment points out that you need to include the format especially for the minutes in this lines.
see:
```java    
private static final DateTimeFormatter minutesFormat = DateTimeFormatter.ofPattern("mm");
```